.aegir.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+module.exports = {
+  // Have not used webpack so just copied proposed setup from
+  // https://babeljs.io/docs/setup/#installation
+  webpack: {
+    module: {
```

Comment:
I think ideally we want to merge this into the aegir flow branch so that we can get inline source maps for tests.

Reply:
With and without the file I don't get the correct line numbers. What about if I remove this completely and having proper line numbers in stack traces of the browser tests will be an open issue?